Configuration/Eras/python/Modifier_run2_JMENanoHerwig7_cff.py

```diff
@@ -0,0 +1,3 @@
+import FWCore.ParameterSet.Config as cms
+
+Modifier_run2_JMENanoHerwig7 = cms.Modifier()
```

Comment:
The convention is to not include the "Modifier" in the name of the modifier, i.e. this would be
```suggestion
run2_JMENanoHerwig7 = cms.Modifier()
```